_posts/en/pages/faq/2016-01-21-rbf-faq.md

```diff
@@ -0,0 +1,237 @@
+
+As a result the vast majority of the security for unconfirmed transactions comes not from within the Bitcoin system, but from factors like the customer's unwilliness to defraud, the vendors tolerance for small amounts of fraud, the existence of the possibility of external recourse, etc. all of which hold true for Opt-In RBF (And not unlike the situation for credit card payments in the US-- they're almost perfectly reversible for _months_). Also, because RBF can sometimes eliminate long confirmation delays, some uses which previously felt forced to accept unconfirmed transaction will no longer need to do so, which reduces their fraud exposure.
+
+However, no one using Opt-in RBF is insisting that you agree with the above view on unconfirmed transaction security. Opt-in RBF is Opt-in, and if it doesn't fit your needs you are not required to use it.
+
+## Who invented unconfirmed transaction replacement? Does it go against the "vision" of Bitcoin?
+
+Transaction replacement for unconfirmed transactions was [a feature in the very first release of Bitcoin](https://github.com/trottier/original-bitcoin/blob/master/src/main.cpp#L434).  Transactions could mark themselves as replaceable by setting a non-maximal sequence number. This was later disabled because it was vulnerable to denial of service attacks.
+
+Although the feature was very useful the denial of service problems and the fact that it was not incentive compatible (What incentive would miners follow the convention and accept a replacement?-- what if they made more from the earlier version?) kept it from being restored.
+
+Later Peter Todd proposed requiring replacements to pay strictly greater feerate and to require that the replacement increase the feerate by at least the minimum required to relay a novel transaction. This both eliminated the denial of service and incentive compatibility problems.
+
+Peter's original work went further and carried the incentive compatibility to its logical conclusion, reasoning that with anonymous, ephemeral, self-selecting miners the only behavior you can really count on is replacement with higher fees. Higher fee preference can also be made more globally convergent than seen first, because there is no global definition of 'first' in an asynchronous distributed system. Because of these reasons and because the system behaving in expected ways is more secure and protective than the system behaving in "unpredictable but better on average ways", he proposed making replacement happen for all transactions. He also proposed a protocol to remove economic gains from double spending in a strong incentive compatible way, called replacement scorched earth-- if someone attempts to double spend you, you send the funds all to fees so the attacker doesn't get them, but it was the kind of proposal only a game theorist could really love. This generated a lot of controversy and subsequently Peter Todd backed his proposal off to the original opt-in behavior.
```

Comment:
You should probably mention here that, after Peter's proposal, we heard a number of wallets call very strongly for opt-in RBF, as they really needed a reasonable way to handle fees and saw opt-in RBF as a good way to do so.